e2e-tests/tfm/cypress/fixtures/mock-gef-facilities.js

```diff
@@ -75,12 +92,13 @@ exports.MOCK_FACILITY_TWO = {
 };
 
 exports.MOCK_FACILITY_TWO_NULL_MIA = {
+  ...getFacilityEndDateProperties(),
   type: CONSTANTS.FACILITY_TYPE.CASH,
   hasBeenIssued: true,
   name: 'Facility two',
   shouldCoverStartOnSubmission: true,
   coverStartDate: 1638403200000,
-  coverEndDate: '2030-01-01T00:00:00.000Z',
+  coverEndDate: dateConstants.tomorrow,
```

Comment:
If this is going to change with the current day, should cover start date & FED also?

Reply:
Also I think in general all  cover start and end dates should be `Date`s in gef facilities

Reply:
I've reverted this for now, I think this falls under techdebt/code improvement for the tests